smarc_bt/src/reconfig_server.py

```diff
@@ -56,6 +56,16 @@ def __init__(self, config):
                                config.ENABLE_MANUAL_MISSION_LOG)
 
 
+        # dubins planner stuff
```

Comment:
A checkbox to completely disable dubins-ing would be very useful for field debugging. Can be enabled by default. 
Also, turning radius is self-explanatory but intersection-radius isnt, maybe add a line before the default bit to explain it? (Adding `dubins_` prefix to both would also be useful to differentiate what they are for)

Reply:
https://github.com/smarc-project/smarc_missions/blob/b123784eb83f64cc18e50cc96c8cf53db9ca1261/smarc_bt/src/mission_plan.py#L262

There's a compute_dubins flag in mission_plan, I could make it dynamic reconfigurable so that it can be changed on the fly

Reply:
dynamic is the way to go :+1: